src/ts/factory/calendarControl.tsx

```diff
@@ -74,6 +83,12 @@ export default abstract class CalendarControl extends EventHandler<ExternalEvent
       internalEvent: this._internalEvent,
       externalEvent: this._externalEvent
     };
+
+    this._renderDate = toStartOfDay();
+    this._renderRange = {
+      start: toStartOfDay(),
+      end: toStartOfDay()
```

Comment:
Will the `end` value change depends on view('week', 'month', 'day') afterwards?

Reply:
It's just initilization at now. It'll be implemented as some code in `Calendar.prototype.move`.